eth/tracers/api.go

```diff
@@ -728,6 +750,13 @@ func (api *API) standardTraceBlockToFile(ctx context.Context, block *types.Block
 	}
 	defer release()
 
+	if beaconRoot := block.BeaconRoot(); beaconRoot != nil {
+		context := core.NewEVMBlockContext(block.Header(), api.chainContext(ctx), nil)
+		vmenv := vm.NewEVM(context, vm.TxContext{}, statedb, api.backend.ChainConfig(), vm.Config{})
+
+		core.ProcessBeaconBlockRoot(*block.BeaconRoot(), vmenv, statedb)
+	}
+
```

Comment:
Please move this down, to right before transactions are iterated. Then you can also use the `chainConfig` which might contain overrides, rather than `api.backend.ChainConfig(),`. It might not make any difference for now, but it is the correct thing to do, IMO.

Reply:
Yep good idea. Applied these changes locally. Will push when I get access.